scratchapi2/user.py

```diff
@@ -1,47 +1,56 @@
-#pylint: disable=trailing-whitespace
 """
 Scratch API User
 
 Anything returned from API does not contain any attributes.
 Use info() to set/get it.
 
+Contains classes:
+* Project
+* User
+* Classroom
 """
-#pylint: enable=trailing-whitespace
 
 import requests
 from .excs import ScratchAPIError
 from .gclass import GenericData
 
+#pylint: disable=too-many-instance-attributes,too-many-function-args
+
+def _request(path, *opts, api_url="https://api.scratch.mit.edu/"):

```

Comment:
This is still defined separately, though, because errors can be raised here. It's helpful to have the exception logic split off as well.